pkg/kapp/resources/mod_field_copy.go

```diff
@@ -74,7 +75,7 @@ func (t FieldCopyMod) apply(obj interface{}, path Path, fullPath Path, srcs map[
 
 		case part.ArrayIndex != nil:
 			if isLast {
-				panic("Expected last part of the path to be map key")
+				return false, errors.New("Expected last part of the path to be map key")
```

Comment:
We generally use `fmt` for creating our errors. Could you make this change?
```suggestion
				return false, fmt.Errorf("Expected last part of the path to be map key")
```

Reply:
@100mik, even if error message has no placeholders?

Reply:
Yep!

Reply:
Done! I didn't use github suggestion because there'd be unused import.

Reply:
Thank you 🥳